### DISCUSSION TITLE
Missing values - OKX grid points

### DISCUSSION THREAD

**Discussion by ghost** [weather-gov/api#541] [Q&A]
I'm seeing (below screenshot) that some of these values are returning empty. I noticed this at least a week or two ago now, I was hoping it would be sorted out by now, but I haven't found any notifications about an outage or service degradation on the official website.

<img width="665" alt="Screen Shot 2022-03-27 at 8 52 45 PM" src="https://user-images.githubusercontent.com/81534044/160309353-0cabbbd8-2bf7-4d88-bbf8-fecc49153b21.png">

**Comment by chaunceygardiner** [comment]
Hourly forecasts don’t have name and detailedForecast filled in.  They are filled in for the twelve hour forecasts.
